### DISCUSSION TITLE
New code view: Can we get the commit box back?

### DISCUSSION THREAD

**Discussion by tripleee** [community/community#49277] [Repositories]
### Select Topic Area

Product Feedback - New Code View beta

### Body

I enabled the "New code view and code search" beta just now, and I generally find it acceptable, though the side pane with the folder view doesn't really bring me any benefits (thanks for being able to hide it).

However, the flow to commit a change has changed in a way which hampers my ability to work in the web UI. I sometimes need to change multiple places in a file, and collect my notes by copy/pasting into the commit message. Now that the "Commit changes ..." dialog only appears once you are done editing and click the button with this title, I can no longer do that. I find that this change slows me down and frustrates me; is there any way we could get it back in the editing view itself?

**Comment by YasinDehfuli** [comment]
First navigate to the file you want to edit.
Second click the pencil icon in the top right corner to open the file for editing.
Third make your changes to the file.
Then once you've made your changes, scroll down to the bottom of the page.
And then You should see a checkbox labeled "Start a commit". Check this box.
Next a commit box will appear where you can enter a commit message and choose the branch you want to commit your changes to.
At All once you're ready to commit your changes, click the green "Commit changes" button.

**Reply by tripleee** [reply]
This is how it used to work, but with the new code view (which is currently in beta) they changed it.
